schedule.js

```diff
@@ -5,11 +5,20 @@ const scheduleConfig = require('./app/lib/scheduleConfig');
 const etl = require('./app/lib/smartEtl');
 const dataService = require('./app/lib/azureDataService');
 
-log.info(`Scheduling job with rule '${scheduleConfig.getSchedule()}'`);
-schedule.scheduleJob(scheduleConfig.getSchedule(), async () => {
-  try {
-    await etl.start(dataService);
-  } catch (ex) {
-    log.error('Unexpected error in service', ex);
-  }
-});
+const nodeEnv = process.env.NODE_ENV;
+
+log.info(`NODE_ENV set to ${nodeEnv}`);
+if (nodeEnv === 'production') {
```

Comment:
Thought I'd added a comment on this but it seems to have been lost.

It's useful to have a way of running the ETL locally without the scheduler, however there's a risk that someone setting 'development' as the NODE_ENV on a Rancher deployment would cause the container to continually restart and keep running the ETL.

The same thing could be achieved by running 'node app.js' rather than 'node schedule.js' in the local docker-compose without additional code.

Reply:
Fair point, however unlikely it is to happen, its bound to happen at some point. I'll revert the changes.